src/wp-admin/js/post.js

```diff
@@ -2,172 +2,244 @@
 /* global theList:true, theExtraList:true, getUserSetting, setUserSetting, commentReply */
 
 var commentsBox, WPSetThumbnailHTML, WPSetThumbnailID, WPRemoveThumbnail, wptitlehint, makeSlugeditClickable, editPermalink;
-// Back-compat: prevent fatal errors
+// Backwards compatibility: prevent fatal errors.
 makeSlugeditClickable = editPermalink = function(){};
 
+// Make sure the wp object exists.
 window.wp = window.wp || {};
 
 ( function( $ ) {
 	var titleHasFocus = false;
 
-commentsBox = {
-	st : 0,
-
-	get : function(total, num) {
-		var st = this.st, data;
-		if ( ! num )
-			num = 20;
-
-		this.st += num;
-		this.total = total;
-		$( '#commentsdiv .spinner' ).addClass( 'is-active' );
-
-		data = {
-			'action' : 'get-comments',
-			'mode' : 'single',
-			'_ajax_nonce' : $('#add_comment_nonce').val(),
-			'p' : $('#post_ID').val(),
-			'start' : st,
-			'number' : num
-		};
-
-		$.post(ajaxurl, data,
-			function(r) {
-				r = wpAjax.parseAjaxResponse(r);
-				$('#commentsdiv .widefat').show();
-				$( '#commentsdiv .spinner' ).removeClass( 'is-active' );
-
-				if ( 'object' == typeof r && r.responses[0] ) {
-					$('#the-comment-list').append( r.responses[0].data );
+	/**
+	 * Control loading of comments on the post and term edit pages
+	 *
+	 * @type {{st: number, get: commentsBox.get, load: commentsBox.load}}
+	 *
+	 * @namespace commentsbox
+	 */
+	commentsBox = {
+		// Comment offset to use when fetching new comments.
+		st : 0,
+
+		/**
+		 * Fetch comments using AJAX and display them in the box
+		 *
+		 * @param {int} total Total number of comments for this post.
+		 * @param {int} num   Optional. Number of comments to fetch, defaults to 20.
```

Comment:
Optional comment should be document using brackets, like this: `@param {int} [num]`